tests/framework/pvc.go

```diff
@@ -62,6 +61,19 @@ func (f *Framework) CreateAndPopulateSourcePVC(pvcName string, podName string, f
 	return sourcePvc
 }
 
+// CreateAndPopulateSourceBlockPVC Creates and populates a PVC using the provided POD and command
```

Comment:
What exactly is this doing? We are creating and populating a block PVC, but this is not called anywhere I can find, and populating a block PVC doesn't make much sense (except maybe for cloning, but that is not part of this PR)